pkg/services/ngalert/models/alert_rule.go

```diff
@@ -79,8 +79,9 @@ const (
 	NamespaceUIDLabel = "__alert_rule_namespace_uid__"
 
 	// Annotations are actually a set of labels, so technically this is the label name of an annotation.
-	DashboardUIDAnnotation = "__dashboardUid__"
-	PanelIDAnnotation      = "__panelId__"
+	DashboardUIDAnnotation    = "__dashboardUid__"
+	PanelIDAnnotation         = "__panelId__"
+	ScreenshotTokenAnnotation = "__alert_screenshot_token__"
```

Comment:
I would stay consistent and use camelCase
```suggestion
	ScreenshotTokenAnnotation = "__alertScreenshotToken__"
```